torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -1008,9 +1050,8 @@ def perspective_bounding_box(
     #   x_out = (coeffs[0] * x + coeffs[1] * y + coeffs[2]) / (coeffs[6] * x + coeffs[7] * y + 1)
     #   y_out = (coeffs[3] * x + coeffs[4] * y + coeffs[5]) / (coeffs[6] * x + coeffs[7] * y + 1)
 
-    numer_denom_points = torch.matmul(points, theta12_T)
-    numer_points = numer_denom_points[:, :2]
-    denom_points = numer_denom_points[:, 2:]
+    numer_points = torch.matmul(points, theta1.T)
+    denom_points = torch.matmul(points, theta2.T)
```

Comment:
Just to understand, previously when we measured this specific change for bboxes it didn't improve the speed?

Reply:
I measured all changes together: concat+single matmul + inplace + aminmax

Reply:
IIUC, it improved the speed for bounding boxes, but the PR didn't measure the impact on images. Since bounding box tensors are usually a lot smaller than images, the perf regression for images was not noticed there.

Reply:
There were no perf regression at all as it was using previous implementation. For bboxes `shape=(1000, 4)` "concat+single matmul" only trick does not bring any speed up on cpu. While working on images (this PR) I see that same trick brings even a slowdown.

Reply:
OK if you measured it now and you know it doesn't slow us down it's fine by me.